Tutorials/NetKetSymmetryInterface.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
<p>Do we still need to manually build the NN graph? <code>Grid</code>  now supports this, as shown e.g. in our paper example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1048/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Yes, I had written this a while ago so I will fix that